OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -553,8 +560,6 @@ constant DebugFlag NF_DUMP_FLAT = DEBUG_FLAG(192, "nfDumpFlat", false,
   Gettext.gettext("Dumps the flat model structure before generating the DAE."));
 constant DebugFlag DUMP_FORCE_FMI_ATTRIBUTES = DEBUG_FLAG(193, "force-fmi-attributes", false,
   Gettext.gettext("Force to export all fmi attributes to the modelDescription.xml, including those which have default values"));
-constant DebugFlag DUMP_FORCE_FMI_INTERNAL_VARIABLES = DEBUG_FLAG(194, "force-fmi-internal-variables", false,
-  Gettext.gettext("Force to export all internal variables (eg: $CSE) to the modelDescription.xml"));
 
```

Comment:
Didn't we test this flag? I think we need test for `none` and `internal` as well.

Reply:
@arun3688 Please open a separate PR for this issue. I will merge this now.

Reply:
@lochel added test cases for checking fmiFilter= none and internal see https://github.com/OpenModelica/OpenModelica/pull/889